app/workers/webhook_transfer.py

```diff
@@ -123,7 +123,7 @@ async def transfer_shift(item: Dict[str, str]) -> str:
     """Transfer the shift to Airtable"""
     MC.set("person")
     attendee_record = ATRecord.from_mobilize_person(item)
-    insert_or_update_record(attendee_record, False)
+    insert_or_update_record(attendee_record, True)
```

Comment:
It's clearer if you use the parameter name when specifying this value:
```suggestion
    insert_or_update_record(attendee_record, insert_only=True)
```